tooling/build/scripts/publishing/index.ts

```diff
@@ -124,12 +124,21 @@ async function writeContentToFile(
       return
     }
 
+    const sanitizedPermalink = path
+      // NOTE: normalization here will remove dual backslashes
+      // and also strip .. filepaths except as a prefix
+      .normalize(fullPermalink)
+      // NOTE: this matches on a leading ../
+      // or a leading ..\
+      // or a plain .. without any paths
+      .replace(/^(\.\.(\/|\\|$))+/, "")
```

Comment:
```suggestion
      .normalize(path.join('./', fullPermalink))
```

Avoid regexes generally because they can be quite brittle, and a str replace is not a 1-for-1 match for how the fs or file read API you pass it to will actually interpret the output string; can't find a problem with regex atm tho! Just a more robust implementation imo.

Reply:
good catch, doing!